Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -1394,4 +1459,24 @@ outputs:
   description: List of URLs that were extracted from the file.
   type: unknown
 tests:
+- Field polling test
```

Comment:
Why did you add all of these tests?

Reply:
Seems like these were added after running demisto-sdk format